src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -310,6 +311,12 @@ private static void syncConfig(boolean load)
         dimensionUnloadQueueDelay = prop.getInt(0);
         prop.setLanguageKey("forge.configgui.dimensionUnloadQueueDelay");
         propOrder.add(prop.getName());
+        
+        prop = config.get(Configuration.CATEGORY_GENERAL, "instantNetworkFlushing", true,
+        		"This is enabled by default as this is vanilla behaivuer, by disabling this packets will only get send after each tick to reduce alot of tiny packets (wich have a very large TCP header)");
```

Comment:
I think the boolean should be inverted and called something like `flushNetworkOnTick = false`.
"When enabled, packets are sent once per tick, which greatly reduces network traffic but may slightly increase latency. When disabled, packets are sent immediately."

We should also consider if we want to enable it by default.

Reply:
Yes this is a good idea, I have implemnted this :)